### DISCUSSION TITLE
怎么实现代码框折叠？

### DISCUSSION THREAD

**Discussion by MengNianxiaoyao** [YunYouJun/valaxy#261] [Q&A]
当代码框内代码过多时，代码框高度就会很大，导致文章长度过长，影响阅读体验。需要能调节代码框高度的设置来控制代码框进行折叠，但我没找到关于此项的主题设置。

**Comment by YunYouJun** [comment]
TODO.

**Comment by MengNianxiaoyao** [comment]
使用了样式控制代码框高度
![image](https://github.com/YunYouJun/valaxy/assets/103258286/0784179d-f779-42e2-93fa-8568f26bce72)
像这样
![image](https://github.com/YunYouJun/valaxy/assets/103258286/e2b77815-b521-41cd-866b-f6ac98dc69d7)
超出的部分就被折叠（类似，可惜不能像butterfly一样有个扩展，只能这样滑动
